apps/apple/FluentUITester/macos/FluentUITester-macOS/FRNViewController.m

```diff
@@ -11,12 +11,18 @@ @interface FRNViewController () <RCTBridgeDelegate>
 
 @implementation FRNViewController
 
-- (void)loadView {
-  // The delegate must conform to RCTBridgeDelegate to retrieve the JS source file for the packager
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  
   RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:nil];
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"FluentUITester" initialProperties:nil];
 
-  [self setView:rootView];
+  NSView *view = [self view];
+
+  [view addSubview:rootView];
+  [rootView setBackgroundColor:[NSColor windowBackgroundColor]];
+  [rootView setFrame:[view bounds]];
+  [rootView setAutoresizingMask:(NSViewMinXMargin | NSViewMinXMargin | NSViewMinYMargin | NSViewMaxYMargin | NSViewWidthSizable | NSViewHeightSizable)];
```

Comment:
Per Chris's comment, can we revert this code which shouldn't be needed and solve the 1px issue in a separate commit?

Reply:
Separate commit or PR? doing so would mean we don't have a usable macOS test app, and that would be a blocker till we either re-apply this fix or come up with a better one.

Reply:
Ideally separate PR - this one seems to be about fixing up install of the test apps so making a separate change to its layout/view hierarchy is surprising